tables/output_tables.Rmd

```diff
@@ -143,19 +143,20 @@ soi_df <- tmb_hist %>%
     ),
     # add Lynch syndrome
     cancer_predispositions = case_when(
-      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "NF-1, Lynch syndrome",
+      Kids_First_Biospecimen_ID == "BS_8AY2GM4G" ~ "Lynch syndrome",
+      sample_id == "7316-515" ~ "Other inherited conditions NOS",
       sample_id == "7316-3058" ~ "Lynch Syndrome",
       sample_id == "7316-2189" ~ "Lynch Syndrome",
       Kids_First_Biospecimen_ID == "BS_85Q5P8GF" ~ "Lynch syndrome",
       TRUE ~ as.character(cancer_predispositions)),
     # add pathogenic germline variants
     `Pathogenic germline variant` = case_when(
-      Kids_First_Participant_ID == "PT_3CHB9PK5" ~ "MSH6",
-      Kids_First_Participant_ID == "PT_0SPKM4S8" ~ "PMS2",
+      Kids_First_Participant_ID == "PT_3CHB9PK5" ~ "MSH6 NM_000179.2 c.3439-2A>G splice (P)",
+      Kids_First_Participant_ID == "PT_0SPKM4S8" ~ "PMS2 p.S46I (LP)",
       Kids_First_Participant_ID == "PT_VTM2STE3" ~ "None detected",
       Kids_First_Participant_ID == "PT_EB0D3BXG" ~ "None detected",
       Kids_First_Participant_ID == "PT_S0Q27J13" ~ "None detected",
-      Kids_First_Participant_ID == "PT_JNEV57VK" ~ "MSH2"
+      Kids_First_Participant_ID == "PT_JNEV57VK" ~ "MSH2 p.A636P (P)"
```

Comment:
This one looks like it's the only one in the spreadsheet with an amino acid change. 
Looking at this all together, I might suggest filling in the three variants as - 

- `"MSH6 c.3439-2A>G splice (P)"`
- `"PMS2 c.137G>T (LP)"`
- `"MSH2c.1906G>C; p.A636P (P)"`

What do you think?

Reply:
Edit, I see your comment that the first one needs the transcript ID!

Reply:
Ok - for the second one above, look at column BO in the excel sheet. For some reason, the tool did not output the protein change, but it does have one 
>PMS2:NM_000535:exon2:c.G137T:p.S46I,PMS2:NM_001322006:exon2:c.G137T:p.S46I,PMS2:NM_001322014:exon2:c.G137T:p.S46I





</body>

</html>